Version 0.6.0 (#176)
dom96/choosenim: changelog.markdown

# Choosenim changelog

## 0.6.0 - TBD

The major new feature is default installation of 64-bit Nim
binaries on Windows.

> 

Reviewer: ```suggestion
Random selection of changes:
```

Author: This is actually all the changes :D

Reviewer: So...
```suggestion
Changes:
```

Author: Done

> * Use git to `choosenim update devel` instead of deleting,

Reviewer: ```suggestion
* Git is now used for `choosenim update devel` instead of deleting,
```

Author: Done

  downloading and bootstrapping from scratch
* Optionally add `~/.nimble/bin` to PATH on Windows on

>   `--firstInstall`

Reviewer: ```suggestion
  when using the `--firstInstall` flag.
```

Author: Done
